ewah/dag_factories/dbt_dag_factory.py

```diff
@@ -51,10 +51,10 @@ def dbt_dags_factory(
         # Allow using cron-style schedule intervals
         assert croniter.is_valid(
             schedule_interval
-        ), "schedule_interval is neither timedelta nor not valid cron!"
+        ), "schedule_interval is not valid a cron expression!"
         catchup = False
         end_date = None
-    else:
+    elif isinstance(schedule_interval, timedelta):
```

Comment:
Same as previous

Reply:
I adjusted it to initialize the parameters with the default values above, before testing the type of `schedule_interval`.

In case it is timedelta, it is going to overwrite those.